OLS-1385: Release Notes for Lightspeed 0.3 release (#88206)
openshift/openshift-docs: modules/ols-0-3-0-release-notes.adoc

// Module included in the following assemblies:
// release_notes/ols-release-notes.adoc

:_mod-docs-content-type: REFERENCE
[id="ols-0-3-0-release-notes_{context}"]
= {ols-long} version 0.3.0

{ols-official} 0.3.0 is now available on {ocp-product-title} 4.15 or later.

[IMPORTANT]
====

> {ols-official} is designed for FIPS. When running on {ocp-product-title} in FIPS mode, it uses the RHEL cryptographic libraries submitted (or planned to be submitted) to NIST for FIPS Validation on only the x86_64, ppc64le, and s390X architectures. For more information about the NIST validation program, see link:https://csrc.nist.gov/Projects/cryptographic-module-validation-program/validated-modules[Cryptographic Module Validation Program]. For the latest NIST status for the individual versions of RHEL cryptographic libraries that have been submitted for validation, see link:https://access.redhat.com/articles/compliance_activities_and_gov_standards#fips-140-2-and-fips-140-3-2[Compliance Activities and Government Standards].

Reviewer: I am not sure why, but the last link (Compliance Activities and Government Standards) redirects me to: `https://access.redhat.com/en/compliance` when I click it. Is that intentional?

> {ols-official} is designed for FIPS. When running on {ocp-product-title} in FIPS mode, it uses the RHEL cryptographic libraries submitted (or planned to be submitted) to NIST for FIPS Validation on only the x86_64, ppc64le, and s390X architectures. For more information about the NIST validation program, see link:https://csrc.nist.gov/Projects/cryptographic-module-validation-program/validated-modules[Cryptographic Module Validation Program]. For the latest NIST status for the individual versions of RHEL cryptographic libraries that have been submitted for validation, see link:https://access.redhat.com/articles/compliance_activities_and_gov_standards#fips-140-2-and-fips-140-3-2[Compliance Activities and Government Standards].

Author: Lightspeed is a layered product that gets published using the standalone method. The source files reside on their branch, and we use a different attributes file than core OCP doc. I have updated the content to use the RHEL attribute. Thanks for noting this!

> {ols-official} is designed for FIPS. When running on {ocp-product-title} in FIPS mode, it uses the RHEL cryptographic libraries submitted (or planned to be submitted) to NIST for FIPS Validation on only the x86_64, ppc64le, and s390X architectures. For more information about the NIST validation program, see link:https://csrc.nist.gov/Projects/cryptographic-module-validation-program/validated-modules[Cryptographic Module Validation Program]. For the latest NIST status for the individual versions of RHEL cryptographic libraries that have been submitted for validation, see link:https://access.redhat.com/articles/compliance_activities_and_gov_standards#fips-140-2-and-fips-140-3-2[Compliance Activities and Government Standards].

Author: done

====

[id="ols-0-3-0-enhancements_{context}"]
== Enhancements

> The following enhancements have been made with {ols-official} 0.3.0:

Reviewer: I see that you use it in all release note versions, but I would maybe consider dropping the present perfect for just present simple to comply with IBM style guide:

```suggestion
The following enhancements are made with {ols-official} 0.3.0:
```

Same suggestion for the rest.

Author: done for these rel notes
